tests/job_runner_test_tsan_suppressions.txt

```diff
@@ -0,0 +1,2 @@
+# Boost lockfree queue triggers the data race warning, which is a known issue.
+race:boost/lockfree/detail/tagged_ptr_ptrcompression.hpp
```

Comment:
I'm surprised you only need to suppress this. No false positive anywhere else in our deps tree?

Reply:
No.